dts/bindings/flash_controller/nxp,iap-f2.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2019, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP (In-Application Programming) flash memory controller for all kinetis and most lpc devices
```

Comment:
seems pedantic but should we consider explicitly listing the "most lpc devices"?

Reply:
OK, makes sense